setup.py

```diff
@@ -202,6 +202,8 @@ def read(*names, **kwargs):
         'toml >= 0.10.0',
         'pkginfo >= 1.5.0.1',
         'dparse >= 0.4.1',
+        'gemfileparser >= 0.6.2',
+        'nose >= 1.3.7',
```

Comment:
Now that we have forked as gemfileparser2, we should leverage this an remove nose. I would rather not add nose at all

Reply:
I will modify gemfileparser2 and use that library here and remove these :)